cdap-app-fabric/src/main/java/co/cask/cdap/app/preview/PreviewStatus.java

```diff
@@ -33,7 +33,8 @@
     COMPLETED,
     DEPLOY_FAILED,
     RUN_FAILED,
-    KILLED
+    KILLED,
+    KILLED_BY_TIMER
```

Comment:
Can it get killed based on the number of records processed? If so will it be different state?